tests/client_test.go

```diff
@@ -489,6 +489,16 @@ func TestStats(t *testing.T) {
 	if err != nil {
 		t.Errorf("Couldn't remove servers: %v", err)
 	}
+
+	locationZones := stats.LocationZones
+	if len(locationZones) < 1 {
+		t.Errorf("location zones are not available")
+	}
+
+	resolvers := stats.Resolvers
+	if len(resolvers) < 1 {
```

Comment:
Lets follow the convention of the rest of this file. No need for transitive variable directly above?
```suggestion
	if len(stats.Resolvers) < 1 {
```